tests/quick_tests/CMakeLists.txt

```diff
@@ -44,9 +44,9 @@ MACRO(make_quicktest test_basename build_name mpi_run)
   DEAL_II_INSOURCE_SETUP_TARGET(${_target} ${build_name})
 
   IF("${mpi_run}" STREQUAL "")
-    SET(_command ./${_target})
+    SET(_command ${_target})
```

Comment:
did you check that this still works under linux?

Reply:
Yes, it did for me.